templates/default/cassandra-env.sh.erb

```diff
@@ -306,7 +306,7 @@ JVM_OPTS="$JVM_OPTS -Djava.rmi.server.hostname=<%= node['cassandra']['jmx_server
 
 # support for metrics reporter:
 <% if node['cassandra']['metrics_reporter']['enabled'] %>
-JVM_OPTS="$JVM_OPTS -Dcassandra.metricsReporterConfigFile=<%= node['cassandra']['conf_dir'] %>/cassandra-metrics.yaml"
+JVM_OPTS="$JVM_OPTS -Dcassandra.metricsReporterConfigFile=cassandra-metrics.yaml"
```

Comment:
The answer lies in `CassandraDaemon`: the file is loaded using `getResource`, so it should work fine (since `conf` is likely always on classpath).